### ISSUE TITLE
"unknown transaction id" logs

### ISSUE THREAD

**Issue by AskAlexSharov** [anacrolix/torrent#721]
Sometime I see logs like:
```
2022/02/04 10:47:26 dispatching packet received on 0.0.0.0:65535: unknown transaction id 22536839
2022/02/04 10:47:26 dispatching packet received on 0.0.0.0:65535: unknown transaction id 22537027
```

version 1.40.0

**Comment by anacrolix** [comment]
Thanks, I'm looking into several logging improvements at the moment.

**Comment by anacrolix** [comment]
I think this was fixed in a324fbfee3f3a9f134a75e15e13bbeb0c3dcefd8.
